test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -511,6 +511,11 @@ class test_ucp_sockaddr : public ucp_test {
         receiver().close_ep_req_free(receiver_ep_close_req);
     }
 
+    static void close_completion(void *request, ucs_status_t status,
+                                 void *user_data) {
+        *(reinterpret_cast<int*>(user_data)) = 1;
```

Comment:
no need for ( ) around reinterpret_cast
also, maybe use true/false instead of 0/1